src/xtgeo/cube/_cube_import.py

```diff
@@ -314,24 +312,23 @@ def import_stormcube(self, sfile):  # pylint: disable=too-many-statements
             "Something when wrong in {}, code is {}".format(__name__, ier)
         )
 
-    self._ilines = np.array(range(1, ncol + 1), dtype=np.int32)
-    self._xlines = np.array(range(1, nrow + 1), dtype=np.int32)
-    self._ncol = ncol
-    self._nrow = nrow
-    self._nlay = nlay
-    self._xori = xori
-    self._xinc = xinc
-    self._yori = yori
-    self._yinc = yinc
-    self._zori = zori
-    self._zinc = zinc
-    self._rotation = rotation
-    self._values = values.reshape((ncol, nrow, nlay))
-    self._yflip = yflip
-    self._traceidcodes = np.ones((ncol, nrow), dtype=np.int32)
-
-
-def import_xtgregcube(self, mfile, values=True):
+    return {
+        "ncol": ncol,
```

Comment:
Why remove ilines/xlines here? Wouldn't the consumer expect to find these keys?

In general, does it make sense to ensure all such imported dicts have the same keys?

Reply:
ilines/xlines were given the default value so there was no need to return any.